app/views.py

```diff
@@ -83,13 +83,38 @@ def logout():
 @app.route('/showAddScript')
 def showAddScript():
 	if session.get('user'):
-		print session['user']
+		#print (session['user'])
 		return render_template('addScript.html',author = session['user'][0])
 	else:
 		return render_template('error.html',error = 'Unauthorized Access')    
 
 
 
+@app.route('/addScript',methods=['POST'])
+def addScript():
+	try:
+		if session.get('user'):
+			_user = session.get('user')[1]
```

Comment:
The _user variable is automatically obtained from the session variable
`_user = session.get('user')[1] `
It is stored there when the user logs in, and if a login event has not occurred, an unauthorized access error will be raised. Did not make sense to allow a separate user field if the login event is necessary anyway. Is that okay?